gui/src/glChartCanvas.cpp

```diff
@@ -4649,7 +4649,7 @@ void glChartCanvas::RenderS57TextOverlay(ViewPort &VPoint) {
         }
 
         // Grow the ViewPort a bit laterally, to minimize "jumping" of text
-        // elements at left side of screen
+        // elements at right side of screen
```

Comment:
This is the original commit: https://github.com/OpenCPN/OpenCPN/commit/7cfd675399f2410345e9bfca6b62ceffc22850a8#diff-1de4c6b4d37f93222849cddcbf31ec1897282ab829bd7533a63e6ff0205d5d24R3693

Since the width of the view port is  increased, that means this is a change for the right side of the screen. Or am I missing something.
I don't see any difference when I temporarily disable the ViewPort expansion.